### ISSUE TITLE
Добавить возможность использовать имеющуюся ИБ (файловая/серверная) с обновлением из хранилища

### ISSUE THREAD

**Issue by ivanmolodec** [firstBitSemenovskaya/jenkins-lib#59]
На пустой базе не возникнет таких ошибок при обновлении ИБ как то неуникальные записи в регистре сведений, не прошедшие обработчики обновления и так далее. Поэтому иногда имеет смысл подключаться к уже существующей базе и проводить проверки и тесты на ней.

**Comment by nixel2007** [comment]
Кажется дубль https://github.com/firstBitSemenovskaya/jenkins-lib/issues/2

**Comment by nixel2007** [comment]
но тут есть описание, так что ту закрою.

**Comment by ivanmolodec** [comment]
Я вроде пробежался по темам ишузов, но не разглядел эту, сорян

**Comment by ivanmolodec** [comment]
> 

Да, я именно про уже развернутую ИБ. Т.е. какой-то настроенный тестовый стенд условно, который не всегда можно сделать просто из dt (копии прода например), могут потребоваться ручные изменения в настройках/данных. Но загрузка из dt это тоже на самом деле важна вещь, про неё был другой ишуз https://github.com/firstBitMarksistskaya/jenkins-lib/issues/40
